packages/e2e-tests/authenticated/passport-01.test.ts

```diff
@@ -1,21 +1,27 @@
+import crypto from "crypto";
+
 import { openDevToolsTab, startTest } from "../helpers";
 import { E2E_USER_1_API_KEY } from "../helpers/authentication";
 import { warpToMessage } from "../helpers/console-panel";
 import { isPassportItemCompleted, showPassport } from "../helpers/passport";
 import { enablePassport } from "../helpers/settings";
 import { addLogpoint, removeAllLogpoints } from "../helpers/source-panel";
-import { resetTestUser, waitFor } from "../helpers/utils";
+import { waitFor } from "../helpers/utils";
 import test, { expect } from "../testFixtureCloneRecording";
 
-test.use({ exampleKey: "doc_rr_console.html" });
+test.use({
+  exampleKey: "doc_rr_console.html",
+  testScope: crypto.randomUUID(),
+  testUser: "frontende2e1@replay.io",
+});
 
 test(`authenticated/passport-01: Time travel`, async ({
   pageWithMeta: { page, recordingId },
-  exampleKey,
+  testScope,
 }) => {
-  await resetTestUser("frontende2e1@replay.io");
-
-  await startTest(page, recordingId, E2E_USER_1_API_KEY);
+  const queryParams = new URLSearchParams();
+  queryParams.set("testScope", testScope!);
+  await startTest(page, recordingId, E2E_USER_1_API_KEY, queryParams);
```

Comment:
Can we update the `startTest` helper methods to _require_ an API key always include a scope argument as well? e.g.

```ts
async function startTest(
  page: Page,
  recordingId: string
): Promise<void>;

async function startTest(
  page: Page,
  recordingId: string,
  apiKey: string,
  testScope: string
): Promise<void>;

async function startTest(
  page: Page,
  recordingId: string,
  apiKey?: string,
  testScope?: string
) {
  // TS will error if this is called with 3 args
}
```